frontend/src/index.css

```
@@ -908,18 +908,105 @@ body.dark .gallery-create__btn,
   align-items: center;
   justify-content: center;
   text-align: center;
-  /* Make hero fill the full viewport height so cover photos are immersive on load */
+  container-type: inline-size;
+  /* Prefer dynamic viewport units to avoid mobile browser chrome resize jank. */
   height: 100vh;
```

Stylelint (17.14.0)

[error] Duplicate property "height" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)